File: openapi/code_samples/javascript/api_v1_logs_authentication_requests/get.js
// List authentication request logs (most recent first)
const res = await fetch(
  `${process.env.SCALEKIT_ENVIRONMENT_URL}/api/v1/logs/authentication/requests?page_size=10&status=SUCCESS`,
  {
    headers: {
      Authorization: `Bearer ${accessToken}`,
    },
  },
)
const data = await res.json()
// data.authRequests, data.next_page_token, data.total_size

File: openapi/code_samples/python/api_v1_logs_authentication_requests/get.py
import os
import requests

# List authentication request logs (most recent first)
res = requests.get(
    f"{os.environ['SCALEKIT_ENVIRONMENT_URL']}/api/v1/logs/authentication/requests",
    headers={"Authorization": f"Bearer {access_token}"},
    params={
        "page_size": 10,
        "status": ["SUCCESS", "FAILED"],
    },
)
data = res.json()
# data["authRequests"], data["next_page_token"], data["total_size"]
